docs/install.rst

```diff
@@ -148,9 +148,17 @@ this case you may consider using the ``--user`` option to install the package
 into your home directory. You can read more about how to do this in the `pip
 documentation <https://pip.pypa.io/en/stable/user_guide/#user-installs>`_.
 
+{% if is_development %}
+
+Alternatively, if you intend to do development on other software that uses
+``astropy``, such as an affiliated package, consider installing ``astropy`` into a :ref:`virtualenv<using-virtualenv>`.
+
+{%else%}
+
 Alternatively, if you intend to do development on other software that uses
-``astropy``, such as an affiliated package, consider installing ``astropy`` into a
-:ref:`virtualenv<using-virtualenv>`.
+``astropy``, such as an affiliated package, consider installing ``astropy`` into a `virtualenv <https://docs.astropy.org/en/latest/development/workflow/virtualenv_detail.html>`__.
```

Comment:
Note: This gives the exact rendered look, except the link when you hover over the text.

Reply:
This is fine to me for now, but it touches on another point I would like to improve in the docs, by including more details (such as how to use virtualenv) in the install docs.  Having this link is maybe ok for now, since even with installation docs improvements there still might be a link like "see here for more details on virtualenv".

OTOH it might be good just to remove this link outright--as I have noted in #10508 this page is outdated and misleading.

Reply:
I see that you have followed up at #11367 . Thanks!